packages/playground/src/weblets/tf_deployment_list.vue

```diff
@@ -235,6 +235,20 @@
           />
         </template>
 
+        <template #StaticWebsite-actions="{ item }">
+          <IconActionBtn
+            tooltip="Show Details"
+            icon="mdi-eye-outline"
+            @click="openDialog(tabs[activeTab].value, item)"
+          />
+          <IconActionBtn
+            tooltip="Visit"
+            icon="mdi-web"
+            color="anchor"
+            :href="'https://' + item.value.env.STATICWEBSITE_DOMAIN"
```

Comment:
It gives bad gateway.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/9de16a6e-4b36-4bff-96a1-5d808d009710)

Reply:
If the instance takes some time to be ready, can we add a hint after we deploy it?

Reply:
can you please provide the the info you used to deploy this instance

Reply:
The repo name I used was capitalized. That's why it failed.

Reply:
Maybe we can add a hint next to the repo name that it's case-sensitive and the deployment won't work unless it's 100% correct?